conf/services/income-tax-sign-up.cy.yml

```diff
@@ -7,118 +7,86 @@ serviceUrl: /report-quarterly/income-and-expenses/sign-up
-  - Nid yw’r cyfieithiadau Cymraeg yn gyflawn. Nid yw’r copi canlynol wedi ei gyfieithu i'r Gymraeg - ‘Ewch yn syth i’r prif gynnwys’ - ‘Defnyddiwch feddalwedd i anfon Diweddariadau Treth Incwm’ (yn nheitl y dudalen a phennawd) - ‘Help’ in ‘Help gyda'r dudalen hon.’ - ‘Help’ in ‘Help wrth ddefnyddio GOV.UK’ - ‘Mae’r holl gynnwys ar gael dan y Drwydded Llywodraeth Agored, fersiwn 3.0, oni nodir yn wahanol’.
-  - Ni all y defnyddiwr gynyddu maint ffontiau ar bob tudalen. Mae’n bosibl y bydd rhai defnyddwyr â nam ar eu golwg yn ei chael hi’n anodd darllen popeth.
-  - Mae’r mewnbwn dyddiad geni yn rhannol anghyflawn, felly ni all defnyddwyr awtolenwi gwerthoedd.
-  - Mae’r sgrin gadarnhau yn defnyddio gwyrdd golau sy’n effeithio ar gyferbyniad lliw, felly gall rhai defnyddwyr â nam ar eu golwg ei chael hi’n anodd darllen popeth ar y dudalen hon.
+  - Mae’n bosibl na fydd y togl iaith y maint neu’r lliw cywir ar rhai sgriniau. Gall hyn fod yn fwy amlwg i bobl sy’n defnyddio darllenwyr sgrin.
+  - Pan fo gwallau ar rhai tudalennau, efallai bydd darllenwyr sgrin yn eu hepgor.
+  - Mae’n bosibl y bydd gwallau lliw gweledol ar gyfer negeseuon gwall ar rhai tudalennau. Mae’n bosibl y bydd rhai defnyddwyr sydd â nam ar eu golwg yn ei chael hi’n anodd darllen popeth.
+  - Mae’n bosibl na fydd rhai cysylltiadau’n agor yn gywir. Gall hyn fod yn fwy amlwg i bobl sy’n defnyddio darllenwyr sgrin.
+  - Ar y dudalen ‘Cadarnhewch eich manylion’, nid yw’r disgrifiadau’n cyd-fynd â’r maes mewnbwn. Bydd hyn fod yn fwy amlwg i bobl sy’n defnyddio darllenwyr sgrin.
+  - Nid yw rhai negeseuon gwall yn ddigon penodol ar gyfer rhai defnyddwyr, felly mae’n bosibl na fyddent yn cael eu deall yn rhwydd.
+  - Mae’n bosibl na fydd ambell i destun awgrym yn cael ei ddarllen allan yn gywir gan ddarllenwyr sgrin.
+  - Ar y dudalen ‘Ar gyfer pa flwyddyn dreth ydych am ddechrau cyflwyno diweddariadau?’, mae’n bosibl na fydd ambell i destun sy’n cael ei ddangos yn amodol yn cael ei ddangos yn gywir. Bydd hyn fod yn fwy amlwg i bobl sy’n defnyddio darllenwyr sgrin.
+  - Ar y dudalen ‘Beth yw eich ffynonellau incwm?’, mae’n bosibl y bydd darllenwyr sgrin yn darllen allan gormod o wybodaeth o ran blychau gwirio.
+  - Mae ambell i ddisgrifiad ar goll o destun awgrym a negeseuon gwall ar sawl tudalen. Bydd hyn yn fwy amlwg i bobl sy’n defnyddio darllenwyr sgrin.
+  - Ar y dudalen ‘Beth yw masnach eich busnes?’, mae ambell i dag ar goll o’r negeseuon gwall. Bydd hyn fod yn fwy amlwg i bobl sy’n defnyddio darllenwyr sgrin.
+  - Efallai na fydd ambell i restr yn dangos yn gywir ar sawl tudalen. Bydd hyn fod yn fwy amlwg i bobl sy’n defnyddio darllenwyr sgrin.
+  - Nid oes rhybudd terfyn amser ar ôl mewngofnodi felly gall amser pobl ddod i ben heb iddynt sylweddoli.
+  - Mae'r cyfieithiadau Cymraeg yn anghyflawn. Nid yw'r testun canlynol wedi'i gyfieithu i'r Gymraeg:
```

Comment:
@scullythornton1 I think the colon at the end is throwing off the yaml parsing. I think without the colon it would parse as you have it, but with the colon it needs a slightly different syntax. I think this should work - but worth running the service locally to check it looks as you would expect 👍 
```
accessibilityProblems
  - first
  - second
    spans multiple lines without a colon
  - >
    third:
    that spans multiple lines with a colon
```

Reply:
thanks, I have removed the colon.